projects/Flightworthy.yaml

```diff
@@ -0,0 +1,12 @@
+Name: Flightworthy
+Description: HTCondor development activities
+Department: Computer Science
+FieldOfScience: Computer Science
+Organization: University of Wisconsin-Madison
+PIName: Miron Livny
+
+ID: '654'
+
+Sponsor:
+  CampusGrid:
+    Name: OSG Connect
```

Comment:
```suggestion
  VirtualOrganization:
    Name: GLOW
```

Reply:
Even if their accounts are on OSG Connect + therefore submitting through the OSG VO?

Reply:
I got the go ahead from Lauren to just call it Flightworthy. Exceptions are occasionally okay. :)

Reply:
I'd rather keep this as `OSG Connect` instead of introducing a new value for a field that I think is ultimately meaningless: https://opensciencegrid.atlassian.net/browse/SOFTWARE-3763

Reply:
Should we drop the Sponsor field for new registrations?

Reply:
I don't think we can until we resolve the aforementioned ticket. My guidance in the interim:

- If they're using a CI Connect submit point, use a `CampusGrid` sponsor reflecting the CI Connect submit point they're associated with
- Otherwise, use the relevant VO